roles/download/defaults/main.yml

```diff
@@ -507,8 +507,10 @@ cinder_csi_plugin_image_tag: "latest"
 cinder_csi_node_driver_registrar_image_repo: "{{ quay_image_repo }}/k8scsi/csi-node-driver-registrar"
 cinder_csi_node_driver_registrar_image_tag: "v1.1.0"
 
-dashboard_image_repo: "{{ gcr_image_repo }}/google_containers/kubernetes-dashboard-{{ image_arch }}"
-dashboard_image_tag: "v1.10.1"
+dashboard_image_repo: "{{ docker_image_repo }}/kubernetesui/dashboard-{{ image_arch }}"
+dashboard_image_tag: "v2.0.0-rc6"
```

Comment:
The [v2.0.0](https://hub.docker.com/layers/kubernetesui/dashboard/v2.0.0/images/sha256-b05326def4ecdfa9380f43b2ebc1642cc213958c8b014298e60d90ad85257a95?context=explore) tag is available we need to switch to that.